cloudify_starlingx/utils.py

```diff
@@ -560,9 +590,10 @@ def handle_cert_in_config(client_config):
             cacert_as_file(cacert)
         client_config['os_cacert'] = cafilename
     else:
+        cacert = None
         cafile = None
         cafilename = None
-    return cafile, cafilename
+    return cacert, cafile, cafilename
```

Comment:
I see you return None from this function on line 558 for example, and when you use it in decorator you expect 3 return values always. it's not a problem?

Reply:
Nope